CONTRIBUTING.md

```diff
@@ -2,31 +2,41 @@
 
 Jest to drugi najważniejszy plik po README w projekcie. Dotyczy on osób, które chcą w sposób aktywny wpłynąć na rozwój naszego forum.
 
-# Tworzenie `issue`
+## Standardy kodu
+
+Tworzony kod PHP **musi przestrzegać standardu [PSR-2](http://www.php-fig.org/psr/psr-2/)** tam, gdzie tylko jest to możliwe - zdajemy sobie sprawę, że w niektórych miejscach będzie to niewykonalne przez już napisany kod Question2Answer. Twój kod musi działać z **PHP w wersji 5.6** - z takiej obecnie korzystamy.
+
+Kod w języku JavaScript **powinien być pisany z użyciem standardu ES6**. Jeśli jednak masz zamiar użyć czegoś, z czym potencjalnie mogą mieć problem nawet najpopularniejsze przeglądarki internetowe, warto wspomnieć o tym w pull requeście, abyśmy mogli razem to przedyskutować.
+
+Należy starać się pisać możliwie najnowszy, bezbłędny i czytelny kod.
```

Comment:
To zdanie nic nie wnosi, raczej nikt się nie stara pisać kodu z błędami. Czytelność wychodzi przy code review.

Reply:
Tak, ale też nikt nie powinien wstawiać kodu, którego samodzielnie nie sprawdził wcześniej. Czytelność to sprawa, o którą dba się od postawienia pierwszego znaku.